src/schema/amd-schema.ts

```diff
@@ -0,0 +1,175 @@
+export const ScreenSchema = {
```

Comment:
Please check if there is need to export this schema as it is only being used in the default one